packages/common/src/wms/utils/index.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  findTagByName,
+  findTagsByName,
+  findTagByPath,
+  findTagsByPath,
+  getAttribute
+} from "xml-utils";
+
+import type { WMS_OUTPUT_FORMAT } from "../types";
+
+import {
+  bboxToString,
+  findName,
+  findAndParseAbstract,
+  findAndParseCapabilityUrl,
+  findTagArray,
+  findTagText,
+  formatUrl,
+  parseName
+} from "../../common";
+
+type Legend = {
+  width: number;
+  height: number;
+  format: string;
+  url: string;
+};
+
+type Style = {
+  name: string;
+  title: string;
+  legends: Legend[];
+};
+
+type WMSLayer = {
+  name: string;
+  namespace?: string | undefined;
+  abstract: string | undefined;
+  keywords: string[];
+  srs: string[];
+  bbox?: Readonly<[number, number, number, number]>;
+  attribution?: {
+    title?: string;
+  };
+  styles: Style[];
+  dates: string[];
+};
+
+// export function findLayerTags(xml: string): { inner: string; outer: string }[] {
```

Comment:
👣 ?

Reply:
I have removed findLayerTags as it is not being used.